requirements.txt

```diff
@@ -8,6 +8,7 @@ psycopg2-binary==2.8.5
 xmltodict==0.11.0
 gspread==3.7.0
 oauth2client==4.1.3
+google-auth=2.6.2
```

Comment:
Oops, typo. Should be `==`.

I'll continue testing after this is fixed, since I'm running a clean setup of redash-uploader with this parsons change.

Reply:
Fixed!